docs-ref-conceptual/azure-cli-sp-tutorial-4.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Work with Azure service principals using a password – Azure CLI | Microsoft Docs
+description: Use service principals with a password to gain control over which Azure resources can be accessed.
+manager: jasongroce
+author: daphnemamsft
+ms.author: daphnema
+ms.date: 09/6/2023
+ms.topic: conceptual
+ms.service: azure-cli
+ms.tool: azure-cli
+ms.custom: devx-track-azurecli, seo-azure-cli
+keywords: azure service principal, create service principal azure, create service principal azure cli
+---
+
+# 4 - Using Service Principal with password-based authentication
```

Comment:
Let's change this to something like `Create a service principal using password-based authentication`.  Also, do a quick check that "service principal" isn't capitalized unless "Service" comes at the beginning of a sentence.